hack/generator/pkg/codegen/code_generator.go

```diff
@@ -86,6 +86,8 @@ func corePipelineStages(idFactory astmodel.IdentifierFactory, configuration *con
 		createStorageTypes(),
 		simplifyDefinitions(),
 
+		markStorageVersion(),
```

Comment:
Should this perhaps be plural? `markStorageVersions()` #conflicted

Reply:
I think singular because at any given time there can only be a single storage version as far as each CRD is concerned?

Reply:
Hmm, I guess it's a matter of perspective. Is cardinality governed by the version or the types?

There's only one version of `microsoft.batch` that is our canonical storage version. 
But there are very many types in that one version.

😁 

Happy for this to stay as it is.